### PR TITLE
Example homeassistant update

### PR DESCRIPTION
**Pull request by Halyul**
Updated the homeassistant script so it can support assigning users to internal regular and admin groups

### CODE REVIEW

File: example_configs/home-assistant.md [hunk 1]
```diff
@@ -16,9 +16,20 @@ homeassistant:
...
     - type: command_line
       command: /config/lldap-ha-auth.sh
-      # Only allow users in the 'homeassistant_user' group to login.
-      # Change to ["https://lldap.example.com"] to allow all users
-      args: ["https://lldap.example.com", "homeassistant_user"]
+      # arguments: [<LDAP Host>, <regular user group>, <admin user group>, <local user group>]
+      # <regular user group>: Only allow users in the <regular user group> (e.g., 'homeassistant_user')
+      # group to login. Users will have the default 'system-users' permission.
+      #
+      # <admin user group>: Allow users in the <regular user group> (e.g., 'homeassistant_user') with
```
Comment (nitnelave) [inline example_configs/home-assistant.md:23]: allow users belonging to both the `<regular user group>` and `<admin user group>`

WDYT?

File: example_configs/home-assistant.md [hunk 2]
```diff
@@ -16,9 +16,20 @@ homeassistant:
...
+      # <admin user group> (e.g., 'homeassistant_admin') to login and have the default 'system-admin' 
+      # permission.
+      #
+      # <local user group>: Users in the <local user group> (e.g., 'homeassistant_local') can only access
+      # homeassistant inside LAN network.
+      #
+      # Change to ["https://lldap.example.com"] to allow all users. All of them will have the default
```
Comment (nitnelave) [inline example_configs/home-assistant.md:30]: How about:
> Only the first argument is required. `["https://lldap.example.com"]` allows all users to log in from anywhere and have 'system-users' permissions.

File: example_configs/lldap-ha-auth.sh [hunk 3]
```diff
@@ -66,5 +66,26 @@ fi
 
 DISPLAY_NAME=$(jq -r .displayName <<< $USER_JSON)
 
+IS_ADMIN=false
+if [[ ! -z "$3" ]] && jq -e '.groups|map(.displayName)|index("'"$3"'")' <<< "$USER_JSON" > /dev/null 2>&1; then
```
Comment (nitnelave) [inline example_configs/lldap-ha-auth.sh:70]: I'm not going to pretend I understand this. I assume you tested it?

**Review by nitnelave — CHANGES_REQUESTED** [review]
Some formatting comments.

File: example_configs/home-assistant.md (hunk 1 above)
Comment (Halyul) [reply example_configs/home-assistant.md:23]: My expression was inaccurate. `<regular user group>` is a list of users that are permitted to access `homeassistant`. Upon that, `<admin user group>` enables the users within `<regular user group>` to be in the admin group.

File: example_configs/lldap-ha-auth.sh (hunk 3 above)
Comment (Halyul) [reply example_configs/lldap-ha-auth.sh:70]: i tested the case where a regular user with the local group and an admin user without the local group logged into homeassistant. the behaviour is correct.

**Review by nitnelave — APPROVED** [review]
Thanks!

**Pull request merged**